batch-nft-reveal/app/components/success/Success.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import {
+  Button,
+  Container,
+  HStack,
+  Text,
+  Link,
+  useClipboard,
+  Tooltip,
+  Center,
+  ButtonGroup,
+} from '@chakra-ui/react'
+import { CopyIcon, ExternalLinkIcon } from '@chakra-ui/icons'
+
+export const Success = (): JSX.Element => {
+  const router = useRouter()
+  const address = '0x4cb093f226983713164a62138c3f718a5b595f73'; // TODO: take the address from url path
+  const network = 'rinkeby'
+  const { hasCopied, onCopy } = useClipboard(address, 3000)
+
+  const viewColleciton = () => {
+    router.push(`/collection/${address}`);
+  }
+
+  return (
+      <Container>
+        <HStack>
+          <Text fontSize="xl">Address:</Text>
+          {address && <Text fontSize="xl">{address}</Text>}
+        <Tooltip
+          label={hasCopied ? 'Copied' : 'Copy to clipboard'}
+          placement="right-start"
+          fontSize="xs"
+          hasArrow
+          closeOnClick={false}
+        >
+          <Link><CopyIcon onClick={onCopy} mx="2px" /></Link>
+        </Tooltip>
+        </HStack>
+        <Center>
+          <HStack>
+            <Link
+              href={`https://${network}.etherscan.io/tx/${address}`}
+              isExternal
+            >
+              Deploy Tx <ExternalLinkIcon mx="2px" />
+            </Link>
+          </HStack>
+        </Center>
+        <Center>
+          <ButtonGroup gap='2'>
+            <Button
+              mt="4"
+              colorScheme="teal"
+              onClick={() => window.open(`https://keepers.chain.link/${network}/new`, '_blank')}
+            >
+              Register Upkeep
```

Comment:
There should be an additional button icon opening the instructions from CL docs. Please check the mockup.